rest/model/host.go

```diff
@@ -30,19 +30,20 @@ type APIHost struct {
 
 // HostPostRequest is a struct that holds the format of a POST request to /hosts
 type HostRequestOptions struct {
```

Comment:
Can you drop a comment to explain that the yaml tags are pulled from CreateHost so we understand why they are different in the future?